content/guides/mobile/ios/quick-start/1-create-ios-app.md

```diff
@@ -0,0 +1,63 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Create an iOS App
+
+Before installing the Box iOS SDK you'll need to have an iOS application
+available to add the dependencies into. We'll create a new blank application
+now.
+
+## Setup an iOS App
+
+<Grid columns='2'>
+  <Choose option='ios.app_setup' value='new' color='blue'>
+    # Create a new iOS application
+
+    If this is the first application you're creating in Xcode or you
+    wish to use a new blank application for testing the SDK, this method
+    will guide you through creating a new blank iOS application.
+  </Choose>
+
+  <Choose option='ios.app_setup' value='existing' color='none'>
+    # Use an existing iOS application
+
+    If you have some iOS development experience and already have a blank
+    iOS application created.
+  </Choose>
+</Grid>
+
+<Choice option='ios.app_setup' value='new' color='blue'>
+  # Create a new iOS application
+
+  1. Load Xcode
+  2. From the top menu, select `File` -> `New` -> `Project...`
```

Comment:
Generally I've been using **Bold** for any text that is a choice and that might need translating.

Reply:
Changed to bold